.github/workflows/build-with-bal-test-graalvm.yml

```
@@ -0,0 +1,184 @@
+name: GraalVM Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      lang_tag:
+        description: Branch/Release Tag of the Ballerina Lang
+        required: true
+        default: master
```

Check all the places.

```suggestion
        default: main
```

---

This is the lang branch used for GraalVM check. Will check other places where we use the default branch name for this module